src/test/java/HomeWork19.java

```diff
@@ -17,14 +19,20 @@ public class HomeWork19 extends BaseTest {
 
     public void deletePlaylistTest() throws InterruptedException {
         loginKoel();
+
         clickSubmit();
 WebElement playlists = getDriver().findElement(By.cssSelector("li[class='playlist playlist']"));
 playlists.click();
-Thread.sleep(1000);
+
+//Thread.sleep(1000);
 deletePlaylist();
-        WebElement successShow = driver.findElement(By.cssSelector("div[class='success show']"));
+        WebDriverWait wait = new WebDriverWait(getDriver(),Duration.ofSeconds(10));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div[class='success show']")));
+        WebElement successShow = getDriver().findElement(By.cssSelector("div[class='success show']"));
 Assert.assertTrue(successShow.isDisplayed());
-    }
+
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)

Reply:
You only received a verification error because you ran the HW19 test separately. There is a bug in Koel that when the playlist is empty, the OK button does not appear when deleting. But if you run all the tests included in TestNG, then such an error will not occur because the playlist will not be empty

Reply:
<img width="1259" alt="image" src="https://github.com/igorpavliuk2/A50-QA-Automation-Framework/assets/152137964/90cdd090-5851-4453-ba71-8780d71e1683">